### DISCUSSION TITLE
10 October Release

### DISCUSSION THREAD

**Discussion by abdfnx** [secman-archive/desktop#42] [Announcements]
## Added

- add password check feature
- create `DeleteBtn` component that we use it in all pages
- add logout auth router
- add secman logo checker
- add the code owners of secman desktop
- add new system api function: `/api/system/import`
- **add Import feature #12**
- **add the secman supporter**, thanks [drift](https://www.drift.com)
   * click this button
      ![sms button](https://user-images.githubusercontent.com/64256993/136689695-eb75eb2e-29a2-4a02-a2dd-88bd1e3f2807.png)
   * then secman supporter will appear
      ![sms](https://user-images.githubusercontent.com/64256993/136689819-8c472fa7-3ded-4fb5-b739-d11607210a74.png)

## Changed

- update secman desktop download website to [**d.secman.dev**](https://d.secman.dev)
- **update Secman Desktop authenticate with cookies**
- make `edit` and `copy` in one component
- update Extra view in all password pages
- update auth login interface
- change authentication URL

## Fixed

- fix secman desktop update checker
- fix Home page, and Email Create page
- fix `CheckPass` component
- fix Server detail page

## New Contributors

* @dependabot made their first contribution in https://github.com/scmn-dev/desktop/pull/17

**Full Changelog**: https://github.com/scmn-dev/desktop/compare/v1.0.2...v1.0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/scmn-dev/desktop/releases/tag/v1.0.3'>10 October Release</a>.</em>

**Comment by david-tomson** [comment]
what a release
